### DISCUSSION TITLE
add templates or awesome page in guide

### DISCUSSION THREAD

**Discussion by xxholly32** [element-plus/element-plus#17248] [Ideas]
个人对于开发一些页面喜欢有个能抄代码地方，其实纯官方的 demo 很多的展示只是组件级别，我个人会去做或者参考很多类似admin-template的网站（当然也可以有一个类似 [awesome-vue](https://github.com/vuejs/awesome-vue) 的仓库），但说实话我在github搜索的结果都没找到特别合适的，最近我也看到 [primeVue templates](https://primevue.org/templates/)，感觉这种模式非常不错，可以拉进社区与真实用户的距离，并能够激发一些用户持续创造的动力；同时可以增强社区互动并扩大用户群体

> 考虑到现在还没搜到一个合适的，感觉我自己也会做一个 admin-template，如果大家有推荐的 admin-template 也可以回复一下，谢谢

Personally, I prefer having a place to copy code when developing some pages. Many official demos are often only at the component level. I tend to create or refer to many admin-template websites (of course, there could also be a repository similar to [awesome-vue](https://github.com/vuejs/awesome-vue)). However, to be honest, I haven't found particularly suitable results on GitHub. Recently, I came across [PrimeVue templates](https://primevue.org/templates), and I think this model is excellent.It can enhance community interaction and expand the user base, inspiring users to continuously create.

> Considering I haven't found a suitable one yet, I feel I might create an admin-template myself. I wonder if there are any interested friends~

**Comment by warmthsea** [comment]
正有此意，我近期制作了一个 awesome-element-plus 这两天会进行公开并进行维护
并且，我计划后面将此仓库添加到 https://github.com/sindresorhus/awesome

**Reply by xxholly32** [reply]
nice，非常期待~，公开请务必通知我，感谢~

**Comment by warmthsea** [comment]
https://github.com/warmthsea/awesome-element-plus

![image](https://github.com/element-plus/element-plus/assets/45450994/5a033a09-b89f-4cc9-8c1d-7bc713b33f96)
